components/molecules/CardHorizontalBarChart/card-horizontal-bar-chart.tsx

```diff
@@ -52,17 +58,22 @@ const CardHorizontalBarChart = ({ languageList, withDescription }: CardHorizonta
         {sortedLangArray.map(({ languageName, percentageUsed }, index) => {
           return (
             index < 5 && (
-              <div
+              <button
                 key={index}
                 onMouseOver={() => handleChangeDescriptText(languageName)}
+                onFocus={() => handleChangeDescriptText(languageName)}
+                onKeyDown={(e) => handleKeyDown(e, languageName)}
+                aria-label={`${languageName}: ${percentageUsed}%`}
```

Comment:
Now that we're using `<span className="sr-only">{`languageName ${percentageUsed}%`}</span>` this is no longer needed. I missed that in my last review.

```suggestion
```